06_gpu_and_ml/hyperparameter-sweep/hp_sweep_gpt.py

```diff
@@ -106,16 +105,20 @@
 
 # We'll spin up a separate container to monitor the training logs with TensorBoard.
 
-monitoring_image = Image.debian_slim(python_version="3.11").pip_install(
+monitoring_image = modal.Image.debian_slim(python_version="3.11").pip_install(
     "tensorboard==2.17.1"
 )
 
-# And we'll deploy a web UI for interacting with our trained models using Gradio.
 
-ui_image = Image.debian_slim(python_version="3.11").pip_install(
-    "gradio~=4.44.0", "pydantic>=2", "fastapi==0.114.2"
+# We'll serve a simple web endpoint
+web_image = modal.Image.debian_slim(python_version="3.11").pip_install(
+    "fastapi[standard]==0.114.2", "pydantic==2.9.1", "starlette==0.41.2"
```

Comment:
I think we need to either bump the `fastapi` version or downgrade the `starlette` version here, see [this CI failure](https://github.com/modal-labs/modal-examples/actions/runs/11582304105/job/32246787164?pr=951).